libraries/voicerecorder/api/src/main/kotlin/io/element/android/libraries/voicerecorder/api/VoiceRecorderState.kt

```diff
@@ -16,9 +16,11 @@
 
 package io.element.android.libraries.voicerecorder.api
 
+import androidx.compose.runtime.Immutable
 import java.io.File
 import kotlin.time.Duration
 
+@Immutable
```

Comment:
Just for my info: What does this do? I mean, what wasn't working well before that prompted you to add this?

Reply:
IIRC it was flagged as a problem when I added `VoiceRecorderState` as a parameter to a composable function [here](https://github.com/vector-im/element-x-android/pull/1758/files#diff-982ecb7d870bf285bd89cdd0681db9f1d98214b4c69a3600591d4e2f8be3bf02R204). It's a class from another module that wasn't built with the compose compilerso the stability couldn't be inferred (see [here](https://developer.android.com/jetpack/compose/performance/stability/fix#multiple-modules) for more info).